src/bci_build/package/spack/README.md.j2

```diff
@@ -0,0 +1,57 @@
+# Spack %%spack_version%% Container Image
+
+Spack is a package manager for supercomputers. It provides build recipies
+for > 6k software components and allows to build entire HPC application
+stacks with little to no prerequisits installed.
+
+This container may be used as a build container when creating a `Dockerfile`
+ or `appainter.def` file using `spack containerize`. It can be used to  run
+ spack commands directly as well. This may require to bind-mount local
```

Comment:
```suggestion
spack commands directly as well. Running `spack` commands directly can require to bind-mount local
```

Reply:
IHMO 'may' expresses exactly what I would like to say.